src/zmq.cpp

```diff
@@ -40,16 +40,14 @@
 #include <poll.h>
 #endif
 
+// TODO: determine if this is an issue, since zmq.h is being loaded from pch.
 // zmq.h must be included *after* poll.h for AIX to build properly
```

Comment:
I wanted to highlight this comment (also). The previous implementation did not conform to the requirement indicated by the comment. I don't have the ability to test this, but this pull request does not change the actual behavior.